pysat/tests/test_files.py

```diff
@@ -121,29 +121,30 @@ def test_parse_delimited_filename(self):
         # Note: Can be removed if future instrument that uses delimited
         # filenames is added to routine travis end-to-end testing
         fname = ''.join(('test_{year:4d}_{month:2d}_{day:2d}_{hour:2d}',
-                         '_{min:2d}_{sec:2d}_v01_r02.cdf'))
+                         '_{mins:2d}_{sec:2d}_v01_r02.cdf'))
         year = np.ones(6)*2009
```

Comment:
To keep some consistency, we could change year to years, month to months, for these variables. This is purely style now.

Reply:
There are arguments for both.  Adding to agenda.

Reply:
An alternative is to just use 'minute', which would be less ambiguous.  Plural works fine for most of the keywords, but 'mins' and 'secs' aren't actually intuitive.

Reply:
Yeah, 'minute' and 'second' could work. All the names are now clear. I prefer short names but 'minute' is better than 'mins'.

Reply:
Updated to minute and second.